### PR TITLE
Lesson 2

### PR DESCRIPTION
**Pull request by vkhmelko-softheme**
add some assets for Lesson 2 (images)

### CODE REVIEW

File: internship2018-1/package.json [hunk 1]
```diff
@@ -4,7 +4,6 @@
   "description": "This is test project for Internship",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
```
Comment (vkhmelko-softheme) [inline internship2018-1/package.json]: Why did you do that?

Comment (vkhmelko-softheme) [reply internship2018-1/package.json]: we don't have tests for now. So it could be removed

Comment (vkhmelko-softheme) [reply internship2018-1/package.json]: Ok! Sounds good for me

Comment (dchecherynda-softheme) [reply internship2018-1/package.json]: 😃

**Pull request merged**